nebula_decoders/src/nebula_decoders_velodyne/decoders/vlp32_decoder.cpp

```diff
@@ -250,6 +281,9 @@ void Vlp32Decoder::unpack(const velodyne_msgs::msg::VelodynePacket & velodyne_pa
 
           intensity = raw->blocks[i].data[k + 2];
 
+          auto block_timestamp = rclcpp::Time(velodyne_packet.stamp).seconds();
```

Comment:
I think this one was already defined earlier in the code. Can you check?

Reply:
You're right! Fixed in 12077a81dd3969d7210baaf1a92440ee132bc096 along with 3 other unnecessary lines of code in the VLS128 decoder. Thanks!